cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -372,6 +384,32 @@ func (uc *UpstreamController) updateNodeStatus(stop chan struct{}) {
 			}
 
 			switch msg.GetOperation() {
+			case model.InsertOperation:
+				klog.Infof("Received register node request, creating node")
+				if _, err := uc.createNode(name); err != nil {
```

Comment:
We should receive the node obj instead of nodename from edge, kubelet has some features(flags like taints/labels) that set to node.